lxd/storage_lvm.go

```diff
@@ -24,7 +24,7 @@ var storageLvmDefaultThinPoolName = "LXDPool"
 func storageLVMCheckVolumeGroup(vgName string) error {
 	output, err := exec.Command("vgdisplay", "-s", vgName).CombinedOutput()
 	if err != nil {
-		shared.Log.Debug("vgdisplay failed to find vg", log.Ctx{"output": string(output)})
+		shared.Log.Error("vgdisplay failed to find vg", log.Ctx{"output": string(output)})
```

Comment:
Looks good to me except for this one.

I know our logging is not very consistent but what I would like to end up with is:

 - Properly handled user input error isn't logged as it is included in the return value (and therefore logged by LXD automatically)
 - ERROR: Should be used for any unexpected failure that isn't caused by bad user input
 - WARN: Should be used when a feature isn't available and causes LXD to skip some code, leading to something functional but partially configured (missing cgroup controlers, kernel features or binaries)
 - INFO: Normal operation logging, containers created, started, stopped, deleted, same for images, ... Basically logging successful user interactions so we have a good timeline in the daemon log.
 - DEBUG: Everything else, trying to stay away from duplicate logging by logging things at the source.

vgdisplay failing in this case is interpreted as the VG not existing, the cause for that is bad user input (typo) so I don't think we should be logging anything here.